sql/system-catalogs/rw-catalog.mdx

```diff
@@ -89,10 +89,10 @@ SELECT name, initialized_at, created_at FROM rw_sources;
 | `rw_ddl_progress`                     | Contains the progress of running DDL statements. You can use this relation to view the progress of running DDL statements. For details, see [Monitor statement progress](/operate/monitor-statement-progress).                                                                                                                                                               |
 | `rw_description`                      | Contains optional descriptions (comments) for each database object. Descriptions can be added with the [COMMENT ON](/sql/commands/sql-comment-on) command and viewed with DESCRIBE or SHOW COLUMNS FROM command.                                                                                                                                                             |
 | `rw_event_logs`                       | Contains information about events, including event IDs, timestamps, event types, and additional information if available.                                                                                                                                                                                                                                                    |
+| `rw_fragments` | Contains information about fragments in the database, including fragment IDs, table IDs, and the parallelism policy that explains how each fragment's parallelism is determined. The policy may be one of the following: <ul><li>`inherit(adaptive)`: inherits adaptive parallelism from its job.</li><li>`override(fixed(n))`: manually set to a fixed value.</li><li>`upstream_fragment([id])`: parallelism determined by upstream fragments, used only in NO_SHUFFLE scenarios.</li><li> `single`: the fragment runs with parallelism 1.</li></ul> |
```

Comment:
This fragment can only have a parallelism of 1 due to the characteristics of the operator.